src/content/1.7/webservice/resources/images.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Images
+---
+
+# Resources for Images
```

Comment:
Empty file ?

Reply:
This file doesn't have synopsis or blank parameters :(